client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+import { registerPlugin } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import './jetpack-plugin-sidebar.scss';
+import JetpackLogo from 'components/jetpack-logo';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
+
+const JetpackPluginSidebar = ( { children } ) => (
+	<Fill>
+		{ children }
+	</Fill>
+);
+
+JetpackPluginSidebar.Slot = () => (
+	<Fragment>
+		<PluginSidebarMoreMenuItem target="jetpack" icon={ <JetpackLogo /> }>
+			{ __( 'Jetpack' ) }
+		</PluginSidebarMoreMenuItem>
+		<PluginSidebar name="jetpack" title={ __( 'Jetpack' ) } icon={ <JetpackLogo /> }>
```

Comment:
Mini-mini nit and totally late to this, but I don't think we translate `Jetpack` elsewhere.

Reply:
Oh, that's right. Good catch 👍 

Fixed in #28988